### PR TITLE
feat(client): adapt OpenAPI about version control

### PR DESCRIPTION
**Pull request by graczhual**

### CODE REVIEW

File: graviti/client/__init__.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 """Client module."""
 
 from graviti.client.branch import create_branch, delete_branch, list_branches
+from graviti.client.catalog import get_catalog
```
Comment (linjiX) [inline graviti/client/__init__.py:8]: shall we mention "OpenAPI" in the commit message ?

File: graviti/client/request.py [hunk 2]
```diff
@@ -7,15 +7,18 @@
 
 import logging
 from typing import Any
+from uuid import uuid4
 
 from requests.models import Response
+from tensorbay.__version__ import __version__
```
Comment (linjiX) [inline graviti/client/request.py:13]: import `graviti.__version__`

File: graviti/client/request.py [hunk 3]
```diff
@@ -7,15 +7,18 @@
...
+from tensorbay.utility import config, get_session
 
 logger = logging.getLogger(__name__)
+PARTIAL_URL = "gatewayv2/tensorbay-open-api/v1"
 
 
-def open_api_do(
-    url: str, access_key: str, method: str, **kwargs: Any  # pylint: disable=unused-argument
-) -> Response:
+def open_api_do(url: str, access_key: str, method: str, **kwargs: Any) -> Response:
```
Comment (linjiX) [inline graviti/client/request.py:21]: we need a separate PR for the implementation of `open_api_do`

**Review by linjiX — CHANGES_REQUESTED** [review]

**Review by AChenQ — APPROVED** [review]

File: graviti/client/branch.py [hunk 4]
```diff
@@ -50,18 +64,88 @@ def list_branches(  # pylint: disable=unused-argument
         offset: The offset of the page.
         limit: The limit of the page.
 
-    Return:
+    Returns:
         The response of OpenAPI.
 
+    Examples:
+        List branches:
```
Comment (linjiX) [inline graviti/client/branch.py:71]: shall we use *italic* for example titles?

**Review by Lee-000 — APPROVED** [review]

File: graviti/client/branch.py (hunk 4 above)
Comment (graczhual) [reply graviti/client/branch.py:71]: yes, but "List branches" and "Get the branch with the given name" here are descriptions.

**Review by linjiX — APPROVED** [review]

**Pull request merged**